index.bs

```diff
@@ -1184,8 +1184,36 @@ spec: encrypted-media-draft; for: EME; urlPrefix: https://w3c.github.io/encrypte
       or APIs but is provides an easier access to some information that can be
       used to fingerprint users.
     </p>
-
-    <section>
+  <section>
+      <h3 id='capabilities-model'>
+        Capabilities Model
+      </h3>
+      <p>
+        This specification supports {{MediaDecodingType}} values of {{file}},
+        {{media-source}} or {{webrtc}} as well as {{MediaEncodingType}}
+        values of {{record}} and {{webrtc}}. 
+      </p>
+      <p>
+        In realtime communications as supported in [[webrtc]], media is
+        transported between peers. Although web sites can play a role in
+        relaying of signaling between peers, they are typically not involved
```

Comment:
Signaling is known in WebRTC land but maybe we should be less specific and mention something like: `exchanging the necessary information to negotiate the media parameters that are common to both user agents`.
Then it is clear that since they need to do the exchange, they get the information.